src/helper.cpp

```diff
@@ -98,4 +102,22 @@ bool AreDoubleApproxEqual(const double &value1, const double &value2, const doub
   return std::fabs(value1 - value2) < epsilon;
 }
 
+bool LookupTransform(const std::string &from_frame,
+                     const std::string &to_frame,
+                     tf2::Transform *tf_output) {
```

Comment:
not sure how I feel about this function. The first time it's called, it won't have any memory of tf topics that were published before the call, since the static variables aren't initialized until first call. So I could see the first call being very long or missing previously-published tfs.